style.css

```diff
@@ -75,3 +87,309 @@ ul {
   left: 3px;
   top: 6px;
 }
+
+.works {
+  /* display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center; */
+  text-align: center;
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  align-content: center;
+  row-gap: 20px;
+}
+
+/* .work-item {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  display: grid;
+  grid-template-rows: 252px 222px;
+} */
+
+.works-heading {
+  grid-row: 1 / 2;
+}
+
+.work-item01 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 2 / 3;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item02 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 3 / 4;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item03 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 4 / 5;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item04 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 5 / 6;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item05 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 6 / 7;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
```

Comment:
- [ ] As required by the previous reviewer, kindly make sure all content inside each card is aligned using `flexbox`. The `Grid` is supposed to be used only on the section container as you did. I suggest, now you align the all card's content using **flexbox** to match the project's requirements.

- [ ] Kindly make sure all the project cards all are centered to stick to the Figma template design. I know you can do it :muscle: :muscle:

Reply:
[@tresorsawasawa](https://github.com/tresorsawasawa)

Reply:
[@tresorsawasawa](https://github.com/tresorsawasawa)
Hi! Thanks for the feedback. I'm not sure I'm understanding the flexbox request. At his moment, the content inside each card, which I have named by the class="work-item" is aligned using `flexbox` as you can see in lines 128, 142, etc.

Reply:
[@tresorsawasawa](https://github.com/tresorsawasawa)
Hi! Thanks for the feedback. I'm not sure I'm understanding the flexbox request. At his moment, the content inside each card, which I have named by the class="work-item" is aligned using `flexbox` as you can see in lines 128, 142, etc., which means that each .work-itemXX is a flexbox container. So... what you mean is that the flex items (.work-content, .work-title, .cards)

Reply:
[@tresorsawasawa](https://github.com/tresorsawasawa)
Hi! Thanks for the feedback. I'm not sure I'm understanding the flexbox request. At his moment, the content inside each card, which I have named by the `class="work-itemXX"` is aligned using `flexbox ` as you can see in lines 128, 142, etc., which means that each .work-itemXX is a flexbox container. 
So... what you mean is that the flex items (.work-content, .work-title, .cards) should also be flex containers at the same time (as a nested flexbox) so that their elements could be flex items. 
Am I right?

Reply:
Hi @CindyDorantes 
I am really sorry for the mistake. But you can make some changes and resubmit again. I think it would be approved

Reply:
[@tresorsawasawa](https://github.com/tresorsawasawa)
Good! I will do it. Thanks again.

Reply:
You are welcome. I am sorry again